src/oas2/transformers/__tests__/params.test.ts

```diff
@@ -70,35 +69,50 @@ describe('params.translator', () => {
     });
 
     it('should translate to simple header param', () => {
-      expect(
-        translateToHeaderParams(
-          {},
-          {
-            'header-name': {
-              description: 'a description',
-              type: 'string',
-            },
+      const params = translateToHeaderParams(
+        {},
+        {
+          'header-name': {
+            description: 'a description',
+            type: 'string',
           },
-        ),
-      ).toMatchSnapshot();
+        },
+      );
+
+      expect(params).toHaveLength(1);
+      expect(params[0]).toMatchSnapshot({
+        schema: {
+          'x-stoplight-id': expect.any(String),
```

Comment:
Because if not provided it gets assigned a random id or something?

Reply:
no, it's not a random id, but I feel like there's no need to test the id here. We have some in other spots that verify the whole HTTP service/operation output.